Src/CSharpier/SyntaxPrinter/TrailingComma.cs

```diff
@@ -0,0 +1,16 @@
+namespace CSharpier.SyntaxPrinter;
+
+internal static class TrailingComma
+{
+    public static Doc Print(SyntaxToken nextSyntaxToken, FormattingContext context)
+    {
+        return
+            context.UsePrettierStyleTrailingCommas
+            && !nextSyntaxToken.LeadingTrivia.Any(o => o.IsDirective)
+            ? Doc.IfBreak(
+                Token.Print(SyntaxFactory.Token(SyntaxKind.CommaToken), context),
+                Doc.Null
+            )
```

Comment:
It is worth adding a couple of tests that validate the trailing comma is added/removed. That would require a test file with the expressions before the comma is added/removed, along with an expected file.

After rescanning the comments in #668 I believe most people agreed this logic was appropriate. IE - do not support for a magic trailing comma that forces something to break.

Reply:
Added some tests with an expected file for the expected behavior